quickstart.sh

```diff
@@ -31,7 +31,10 @@ if test -f env/bin/activate
 then
 	source env/bin/activate;
 	pip3 install -r requirements.txt;
-else
+elif [ -v $VIRTUALENV_ENV ]
+then
+  pip3 install -r requirements.txt;
```

Comment:
@YPCrumble Could you adjust the line indent to be further in for consistency?

Reply:
@gregelin ah thank you!  My editor was set so that I didn't realize this had tabs vs. spaces. Just updated and removed spaces lower down in the file for consistency. Let me know if that isn't what you were looking for and I can revert adjust as appropriate. 

Another option would be to convert this to spaces as it seems most of the rest of this repo project doesn't use tabs, though I didn't do that as it would make inspecting the commit history more difficult.

Looking forward to speaking with you today!